packages/react/src/avatar/Avatar.tsx

```diff
@@ -32,7 +30,7 @@ export const Avatar = forwardRef<HTMLSpanElement, AvatarProps>(
     {
       children,
       className,
-      colorScheme = "neutral",
+      colorScheme = "brand",
```

Comment:
let's keep the default as neutral 
```suggestion
      colorScheme = "neutral",
```